home/.chezmoi.yaml.tmpl

```diff
@@ -4,7 +4,7 @@
 {{- $chezmoiForce := or (has "--force" .chezmoi.args) (has "--force=true" .chezmoi.args) -}}
 {{- $interactive := and stdinIsATTY (not $chezmoiForce) -}}
 
-{{- $ubuntu := and .chezmoi.osRelease.ubuntuCodename true -}}
```

Comment:
@twpayne I think this is probably a backwards compatibility break in chezmoi 2.26.0. Just FYI.

Reply:
Is this still an issue? I see that you marked this as resolved. What was the backwards compatibility break?

Reply:
I marked as resolved because the fix is very simple, but my CI pipeline started to fail with no change, except for chezmoi's new version.

But it's ok, I'll check it with a little more detail next week.

Reply:
Thanks for the fast reply! Please open an issue if you find a problem.